pkg/promotion/promotion.go

```diff
@@ -36,6 +43,21 @@ func BuildsOfficialImages(configSpec *cioperatorapi.ReleaseBuildConfiguration) b
 	return RefersToOfficialImage(promotionName, promotionNamespace)
 }
 
+func isWhiteListed(org, repo string, whitelist WhitelistConfig) bool {
```

Comment:
This is basically a method, so why not make it one: `func (w *WhitelistConfig) has(org, repo string) bool`